imagetest/test_suites/oslogin/README.md

```diff
@@ -0,0 +1,36 @@
+# Test Setup
```

Comment:
Is this everything needed to set up oslogin testing? Is a separate test org required and does it need any specific policies?

Reply:
I believe so, yes. A separate test org is not technically required. It does not need any specific policies, either, outside of allowing the person doing the setup to create secrets and provide IAM permissions to the test users. A test org would, however, have the advantages of 1) a clean environment, and 2) being able to easily the test users required. I can include these extra details in the setup page.

Reply:
Done.